src/Gaufrette/Adapter/Ftp.php

```diff
@@ -378,30 +378,27 @@ private function isDir($directory)
             return true;
         }
 
-        $chDirResult = false;
         try {
+            $chDirResult = false;
```

Comment:
This line can be removed as the boolean value is set by the call to `ftp_chdir` :

> Returns TRUE on success or FALSE on failure.

https://www.php.net/manual/en/function.ftp-chdir.php